recipes/foamalgo/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+pip install -v .
```

Comment:
I believe you can remove this file and just add the ``pip`` command in the recipe and it should be fine

Reply:
Done! Thanks!